src/acceptance-test/java/org/zalando/nakadi/webservice/EventPublishingAT.java

```diff
@@ -0,0 +1,36 @@
+package org.zalando.nakadi.webservice;
+
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.response.Response;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.HttpStatus;
+import org.junit.Test;
+import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.webservice.utils.NakadiTestUtils;
+
+import java.text.MessageFormat;
+
+import static com.jayway.restassured.RestAssured.given;
+
+public class EventPublishingAT extends BaseAT {
```

Comment:
Removed in https://github.com/zalando/nakadi/pull/515/commits/b9c36f89d2afabe292c43f0a284b6b4f9c4eb7a7